src/js/coblocks-events.js

```diff
@@ -0,0 +1,76 @@
+( function( $ ) {
+	'use strict';
+	$( document ).ready( function() {
+		observer.observe( document.body, {
+			childList: true,
+			subtree: true,
+			attributes: false,
+			characterData: false,
+		} );
+	} );
+
+	const MutationObserver = window.MutationObserver || window.WebKitMutationObserver;
+
+	const observer = new MutationObserver( function( mutations ) {
+		mutations.forEach( function( mutation ) {
+			if ( ! mutation.addedNodes ) {
+				return;
+			}
+
+			for ( let i = 0; i < mutation.addedNodes.length; i++ ) {
+				// do things to your newly added nodes here
+				const node = mutation.addedNodes[ i ];
+
+				if ( node.className === 'coblocks-ical-events' ) {
+					initializePagination();
+				}
+			}
+		} );
+	} );
+
+	function initializePagination() {
+		const moreEvents = $( '.wp-block-coblocks-events__more-events-wrapper' );
```

Comment:
We should generate the more link just like we are generating the next page and previous page elements.